llpc/test/shaderdb/multiple_inputs/GlslTwoStages.multi-input

```diff
@@ -2,15 +2,33 @@
 
 ; BEGIN_SHADERTEST
 ; RUN: amdllpc -spvgen-dir=%spvgendir% -v \
+; RUN:      -enable-part-pipeline=0 \
 ; RUN:      %S/test_inputs/Vs1.vert \
 ; RUN:      %S/test_inputs/Fs1.frag \
-; RUN: | FileCheck -check-prefix=SHADERTEST %s
+; RUN: | FileCheck -check-prefix=SHADERTEST_PP0 %s
 ;
-; SHADERTEST-LABEL: {{^//}} LLPC final pipeline module info
-; SHADERTEST:       define dllexport amdgpu_vs void @_amdgpu_vs_main
-; SHADERTEST:       define dllexport amdgpu_ps { <4 x float> } @_amdgpu_ps_main
-; SHADERTEST-LABEL: {{^//}} LLPC final ELF info
-; SHADERTEST-LABEL: _amdgpu_vs_main:
-; SHADERTEST-LABEL: _amdgpu_ps_main:
-; SHADERTEST-LABEL: {{^=====}} AMDLLPC SUCCESS ====
+; SHADERTEST_PP0-LABEL: {{^//}} LLPC final pipeline module info
+; SHADERTEST_PP0:       define dllexport amdgpu_vs void @_amdgpu_vs_main
+; SHADERTEST_PP0:       define dllexport amdgpu_ps { <4 x float> } @_amdgpu_ps_main
+; SHADERTEST_PP0-LABEL: {{^//}} LLPC final ELF info
+; SHADERTEST_PP0-LABEL: _amdgpu_vs_main:
+; SHADERTEST_PP0-LABEL: _amdgpu_ps_main:
+; SHADERTEST_PP0-LABEL: {{^=====}} AMDLLPC SUCCESS ====
+; END_SHADERTEST
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v \
+; RUN:      -enable-part-pipeline=1 \
+; RUN:      %S/test_inputs/Vs1.vert \
+; RUN:      %S/test_inputs/Fs1.frag \
+; RUN: | FileCheck -check-prefix=SHADERTEST_PP1 %s
+;
+; SHADERTEST_PP1-LABEL: {{^//}} LLPC final pipeline module info
+; SHADERTEST_PP1:       define dllexport amdgpu_ps { <4 x float> } @_amdgpu_ps_main
+; SHADERTEST_PP1-LABEL: {{^//}} LLPC final pipeline module info
+; SHADERTEST_PP1:       define dllexport amdgpu_vs void @_amdgpu_vs_main
```

Comment:
What's the difference between this and the previous test? The order of functions in IR?
I'm wondering if we can use the same `SHADERTEST` checks if we use `SHADERTEST-DAG` for both `RUN`s. Also in the other tests.

Reply:
Yes, the order of the function is different (and I like the fact that the order is stated explicitly in this test as it shows that there is a difference).